ui/src/app/pages/admin/admin-user-common.tsx

```diff
@@ -160,43 +170,81 @@ export const getUpdatedProfileValue = (
   }
 };
 
+const getModuleStatus = (profile, moduleName) =>
+  computeRenewalDisplayDates(getAccessModuleStatusByName(profile, moduleName))
+    .moduleStatus;
+
+const moduleStatusStyle = (moduleStatus) =>
+  cond(
+    [
+      moduleStatus === AccessModulesStatus.EXPIRED,
+      () => commonStyles.expiredModule,
+    ],
+    [
+      moduleStatus === AccessModulesStatus.EXPIRING_SOON,
+      () => commonStyles.expiringSoonModule,
+    ],
+    () => commonStyles.completeModule
+  );
+
+const displayModuleStatusAndDate = (
+  profile: Profile,
+  moduleName: AccessModule,
+  child: string
+): JSX.Element => {
+  return (
+    <div style={moduleStatusStyle(getModuleStatus(profile, moduleName))}>
+      {child}
+    </div>
+  );
+};
+
 export const isBypassed = (
   profile: Profile,
   moduleName: AccessModule
 ): boolean =>
   !!getAccessModuleStatusByName(profile, moduleName)?.bypassEpochMillis;
 
-export const displayModuleStatus = (
-  profile: Profile,
-  moduleName: AccessModule
-): string => {
-  return computeRenewalDisplayDates(
-    getAccessModuleStatusByName(profile, moduleName)
-  ).moduleStatus;
-};
-
 // Some modules may never expire (eg GOOGLE TWO STEP NOTIFICATION, ERA COMMONS etc),
 // in such cases set the expiry date as NEVER
 // For other modules display the expiry date if known, else display '-' (say in case of bypass)
 const getNullStringForExpirationDate = (moduleName: AccessModule): string =>
   getAccessModuleConfig(moduleName).canExpire ? '-' : 'Never';
 
+export const displayModuleStatus = (
+  profile: Profile,
+  moduleName: AccessModule
+): JSX.Element =>
+  displayModuleStatusAndDate(
+    profile,
+    moduleName,
+    getModuleStatus(profile, moduleName)
+  );
+
 export const displayModuleCompletionDate = (
   profile: Profile,
   moduleName: AccessModule
-): string =>
-  formatDate(
-    getAccessModuleStatusByName(profile, moduleName)?.completionEpochMillis,
-    '-'
+): JSX.Element =>
+  displayModuleStatusAndDate(
+    profile,
+    moduleName,
+    formatDate(
+      getAccessModuleStatusByName(profile, moduleName)?.completionEpochMillis,
+      '-'
+    )
   );
 
 export const displayModuleExpirationDate = (
   profile: Profile,
   moduleName: AccessModule
-): string =>
-  formatDate(
-    getAccessModuleStatusByName(profile, moduleName)?.expirationEpochMillis,
-    getNullStringForExpirationDate(moduleName)
+): JSX.Element =>
+  displayModuleStatusAndDate(
```

Comment:
What do you think of this alternative?
```
<div style={moduleStatusStyle(profile,moduleName}}>
  {formatDate(
      getAccessModuleStatusByName(profile, moduleName)?.expirationEpochMillis,
      getNullStringForExpirationDate(moduleName)
  )}
</div>

Reply:
rather than the function displayModuleStatusAndDate returning div? I  had thought about that, but found this to be neater rather than repetitive code